src/metadata/ColumnMetadata.ts

```diff
@@ -238,6 +239,18 @@ export class ColumnMetadata {
      */
     isVirtual: boolean = false
 
+    /**
+     * Indicates if column is calculated. Virtual columns are not mapped to the entity.
+     */
+    isVirtualDecorator: boolean = false
```

Comment:
"virtual decorator" isn't the best name actually... first there is a confusion between "calculated" and "virtual decorator" now in the naming / description. second virtuals can be defined in entity schemas as well (alternative to decorators). I do understand we already have "isVirtual" here which used for different purpose... that's why I suggest to go with "isVirtualProperty" instead.

Reply:
Sounds good. I will update this in a few hours.

Reply:
This has been updated